src/scripts/add_object_label.py

```diff
@@ -0,0 +1,41 @@
+from typing import TextIO
```

Comment:
Looks simple and like it does the job! I'm guessing that when the [`fill-table`](https://github.com/INCATools/ontology-access-kit/issues/313) issue you opened is fixed, this may be changed from a Python script to an sssom CLI call, as it looks like you suggested.